src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1518,19 +1525,19 @@ private void InitializeIncludedNodes()
             //WhenHomeWorkspaceIsFocusedInputAndOutputNodesAreMissingFromSearch
             //WhenStartingDynamoInputAndOutputNodesAreNolongerMissingFromSearch
             // New index process from Lucene, adding missing nodes: Code Block, Input and Output
-            //var ele = AddNodeTypeToSearch(outputData);      
-            //if (ele != null)
-            //{
-            //    AddNodeTypeToSearchIndex(ele, iDoc);
-            //}
-
-            //ele = AddNodeTypeToSearch(symbolData);
-            //if (ele != null)
-            //{
-            //    AddNodeTypeToSearchIndex(ele, iDoc);
-            //}
-
-            var ele = AddNodeTypeToSearch(cbnData);
+            var ele = AddNodeTypeToSearch(outputData);      
```

Comment:
just consider (for future bug fixes) that uncommenting this will probably generate duplicates in output and input nodes when searching in the Library (due that are already added in library).

Reply:
Thanks @RobertGlobant20. Will test this out.